src/one_dragon/base/operation/application/application_factory.py

```
@@ -14,14 +14,20 @@ class ApplicationFactory(ABC):
 
-    def __init__(self, app_id: str):
+    def __init__(
+        self,
+        app_id: str,
+        app_name: str,
+    ):
         """
         初始化应用工厂。
 
         Args:
             app_id: 应用唯一标识符，用于区分不同的应用类型
```

Ruff (0.13.1)

Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)